driver-core/src/test/java/com/datastax/driver/core/ProtocolVersionRenegotiationTest.java

```diff
@@ -53,45 +62,31 @@ public void should_fail_when_renegotiation_disabled_3_10() throws Exception {
      * @jira_ticket JAVA-1367
      */
     @Test(groups = "short")
-    @CCMConfig(version = "2.1.16", createCluster = false)
-    public void should_fail_when_initial_version_is_beta() throws Exception {
-        Cluster cluster = connectWithBetaVersion();
-        assertThat(actualProtocolVersion(cluster)).isEqualTo(V3);
-    }
-
-    /**
-     * @jira_ticket JAVA-1367
-     */
-    @Test(groups = "short")
     @CCMConfig(version = "1.2.19", createCluster = false)
-    public void should_renegotiate_and_initial_version_too_high_1_2_19() throws Exception {
-        Cluster cluster = connectWithVersion(V4);
-        assertThat(actualProtocolVersion(cluster)).isEqualTo(V1);
-        cluster = connectWithoutVersion();
-        assertThat(actualProtocolVersion(cluster)).isEqualTo(V1);
+    public void should_fail_when_version_provided_and_too_high() throws Exception {
+        UnsupportedProtocolVersionException e = connectWithUnsupportedVersion(V4);
+        assertThat(e.getUnsupportedVersion()).isEqualTo(V4);
+        // pre-CASSANDRA-11464: server replies with its own version
+        assertThat(e.getServerVersion()).isEqualTo(V1);
     }
 
     /**
      * @jira_ticket JAVA-1367
      */
     @Test(groups = "short")
-    @CCMConfig(version = "2.0.17", createCluster = false)
-    public void should_renegotiate_and_initial_version_too_high_2_0_17() throws Exception {
-        Cluster cluster = connectWithVersion(V4);
-        assertThat(actualProtocolVersion(cluster)).isEqualTo(V2);
```

Comment:
There should be no negotiation when an explicit version is provided. `withProtocolVersion`'s javadocs say:
> This method allows to force the use of a particular protocol version.